diff --git a/assets/meson.build b/assets/meson.build
index b746281..e69d1ff 100644
--- a/assets/meson.build
+++ b/assets/meson.build
@@ -17,7 +17,7 @@ foreach file : embedded_assets_f
     input: file,
     output: '@0@.xxd'.format(embedded_assets[count]),
     command: [
-        xxd, '-i', '@INPUT@'
+        xxd, '-i', '-n', '___assets_@PLAINNAME@', '@INPUT@'
     ],
     capture: true,
     depend_files: embedded_assets_f[count]
diff --git a/shader/meson.build b/shader/meson.build
index 4b5e249..9d2390a 100644
--- a/shader/meson.build
+++ b/shader/meson.build
@@ -36,7 +36,7 @@ foreach file : embedded_shaders_f
         input: file,
         output: '@0@.xxd'.format(embedded_shaders[count]),
         command: [
-            xxd, '-i', '@INPUT@'
+            xxd, '-i', '-n', '___shader_@PLAINNAME@', '@INPUT@'
         ],
         capture: true,
         depend_files: embedded_shaders_f[count]
